### DISCUSSION TITLE
Am I disabling the plugins right?

### DISCUSSION THREAD

**Discussion by mehov** [gatsbyjs/gatsby#34485] [Help]
What I want:
--

Temporarily disable `gatsby-plugin-react-helmet`, `gatsby-transformer-sharp`, `gatsby-plugin-sharp` and see what happens

What I tried:
--

Comment out these plugins in `gatsby-config.js`:

```
    /**
     * The following two plugins are required if you want to use Gatsby image
     * See https://www.gatsbyjs.com/docs/gatsby-image/#setting-up-gatsby-image
     * if you're curious about it.
     */
    //`gatsby-transformer-sharp`,
    //`gatsby-plugin-sharp`,
```

```
    // See https://www.gatsbyjs.com/plugins/gatsby-plugin-react-helmet/?=gatsby-plugin-react-helmet
    //`gatsby-plugin-react-helmet`,
```

What happens:
--

> verbose Found 43 changed pages
> success Checking for changed pages - 0.012s
> success Cleaning up stale page-data - 0.014s
> success onPreExtractQueries - 0.003s
> ⠸ extract queries from components
> [==============              ]   4.181 s 1/2 50% Running gatsby-plugin-sharp.IMAGE_PROCESSING jobs
> Killed

What I expected to happen:
--

Not seeing anything related to any of these plugins?

------

I have started the build right after saving the `gatsby-config.js` file. Maybe I should rerun `npm ci`? Or actually delete these plugins from the `npm`, and remove them from `package.json` first?

**Comment by rbruggem** [comment]
I am experiencing the same when I disable `gatsby-plugin-react-helmet`
